src/main/java/com/sap/ordermanegergreen/controllers/CompanyController.java

```diff
@@ -1,9 +1,9 @@
-package com.sap.ordermanegergreen.Controllers;
+package com.sap.ordermanegergreen.controllers;
 
-import com.sap.ordermanegergreen.DTO.TokenDTO;
-import com.sap.ordermanegergreen.Models.Company;
-import com.sap.ordermanegergreen.Services.CompanyService;
-import com.sap.ordermanegergreen.Utils.JwtToken;
+import com.sap.ordermanegergreen.dto.TokenDTO;
+import com.sap.ordermanegergreen.models.Company;
+import com.sap.ordermanegergreen.services.CompanyService;
+import com.sap.ordermanegergreen.utils.JwtToken;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
```

Comment:
No need API for company

the company should be created once by first signup and that it's